index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+      integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="icon" href="./assets/coding.png" type="image/x-icon" />
+  </head>
+  <body>
+    <header>
+      <a id="logo" href="#">Welcome</a>
+      <div class="nav-desktop"></div>
+      
+      <div class="nav-mobile">
+        <nav class="btn"><i class="fa-solid fa-bars"></i></nav>
+      </div>
+    </header>
+
+    <section class="welcome-section">
+      <div class="welcome-header-text">
+        <h1>Hey there. I'm Kano</h1>
+        <h2 class="second-header">I am software developer</h2>
+      </div>
+      <div class="intro-text">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-links">
+         <li>
+          <img
+          src="./assets/social-media/twitter.svg"
+          alt="twitter social media icon "
+        />
+         </li>
+          <li>
+            <img
+            src="./assets/social-media/Linkedin.svg"
+            alt="github social media icon "
+          />
+          </li>
+         <li>
+        <img
+        src="./assets/social-media/medium.svg"
+        alt="Medium social media icon "
+      /></li>
+      <li>
```

Comment:
- [ ] Good job implementing the requested changes by the previous reviewer! you're almost there but I think, It would be better if you add an `<a>` tag with a link to each social media icon.  let's rock, kindly fix it ⚙️

Reply:
I will enclose the icons inside a tags, but i don't see why I should link them to my social media pages after I already did that in the readme file. Besides that i don't have all those accounts represented by those icons.

Reply:
I feel that part is optional please, no part in the instructions mentions that